src/app/config/config.ts

```diff
@@ -155,8 +156,56 @@ const mailConfig: MailConfig = (function () {
     transporter = nodemailer.createTransport(options)
   } else {
     transporter = nodemailer.createTransport({
-      port: prodOnlyVars.port,
-      host: prodOnlyVars.host,
+      port: prodOnlyVars.port_us,
+      host: prodOnlyVars.host_us,
+      ignoreTLS: true,
+    })
+  }
+
+  return {
+    mailFrom,
+    official,
+    mailer,
+    transporter,
+  }
+})()
+
+// Use this mail config when SES migration is over, after renaming env vars back to without _sg suffix (opengovsg/formsg-private#130)
```

Comment:
Add TODO(#130) prefix, so that it is easier to find all the TODOs in the codebase
```suggestion
// TODO(#130): Use this mail config when SES migration is over, after renaming env vars back to without _sg suffix (opengovsg/formsg-private#130)
```

Reply:
no wonder, I was thinking it looked a bit odd but couldn't figure out why. thanks! [d167864](https://github.com/opengovsg/FormSG/pull/5369/commits/d167864734f46ad3fa204bc4db5d370b8bcc6ce8)